tests/server.py

```diff
@@ -474,6 +476,19 @@ def getHostByName(
                     return fail(DNSLookupError("OH NO: unknown %s" % (name,)))
                 return succeed(lookups[name])
 
+        # In order for the TLS protocols to use the proper reactor's clock, patch
+        # _get_default_clock on newer Twisted versions.
+        #
+        # This is *super* dirty since it is never stopped and relies on the next
+        # test to patch over it.
+        #
+        # Use create=True for backwards compatibilty with Twisted <= 23.8.0.
+        if twisted.version > Version("Twisted", 23, 8, 0):
```

Comment:
It feels janky that we check `>` here and `<=` to above... as if there *is* at `23.8.x` then it'll enable one bit of code and not the other.

It sounds like a Twisted 23.1x.0 will be out soon-ish though.

Reply:
23.10, according to https://mail.python.org/archives/list/twisted@python.org/thread/B7FQO3BIAGPFLQMAOALUZQ2EX3AQDPT6/#4FADHHQ5H4U4BZ6LGWI5PV5K7MBOGK2V

Reply:
I can use `>= 23.10.0` if we want (in both places), but it'll fail until the release is out.

Reply:
I think 23.8 is fine, just a note for completeness.